content/githubactions/datacode/_index.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Add Data Code"
+date = 2018-12-08T18:38:38-05:00
+weight = 7
+pre = "<b>7. </b>"
++++
+
+1. Add any packages that your data cleaning and manipulation requires to the `DESCRIPTION` file in your package's root. Note: unfortunately you cannot add the `tidyverse` umbrella package as this will cause the package check to fail; you need to install any `tidyverse` packages you want to use individually. For example, we've just added the `dplyr` and `readr` packages:  
```

Comment:
> Note: unfortunately you cannot add the `tidyverse` umbrella package as this will cause the package check to fail; you need to install any `tidyverse` packages you want to use individually.

I'd suggest a version of this that explains why this issue occurs. Maybe an edit like this:

> Note: Installing R packages can be slow and so you should only install the specific packages you need, not umbrella packages like `tidyverse` (in fact installing `tidyverse` will cause the system to timeout and therefore fail)

FYI - it is possible to get around this issue by using the RStudio Package Manager which installs binaries on Linux. Seek e.g.,

* https://github.com/weecology/portalcasting/commit/669001ca87cd1bb1f77a38d5589ba6830de0c0d2
* https://github.com/weecology/PortalData/blob/main/.github/workflows/main.yml

Reply:
Thanks for the tip. I'll add the simple explanation tho